docs/streams/upgrade-guide.html

```diff
@@ -75,7 +75,21 @@ <h1>Upgrade Guide and API Changes</h1>
     <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API changes in 2.4.0</a></h3>
 
     <!-- Placeholder KIP-213 -->
-    <!-- Placeholder KIP-307 -->
+    <p>
+        In the 2.4 release users now can name all operators in a Kafka Streams DSL topology via
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-307%3A+Allow+to+define+custom+processor+names+with+KStreams+DSL">KIP-307</p>. 
+        The internally generated names on smaller/simple topologies viewing the topology description 
+        generated from executing <code>Topology#describe()#toString()</code> is easy enough to understand, 
+        when the topologies are complex having meaningful names is essential to 
+        understand the full context of what your Kafka Streams application is doing.
+
```

Comment:
If you want  a new line, you need to insert `<br/>` markup?

Reply:
ack